config.py

```diff
@@ -1,21 +1,57 @@
+import os
+import logging
+
+def instance_id_path():
+    ID = '0x1' # default value
+    for file in os.listdir(os.getcwd()):
+        if file.endswith(".log") and not file == "log.log":  # ignore base .log file
+            try:
+                ID = hex(int(file[:-4],16) + 1) # remove the extension, increment by 1, convert to hex
+                os.remove(file)  # if exception not raised, then delete file
+            except:
+                continue  # check the next file
+
+    ID_path = './' + ID + '.log'
+
+    return ID_path
+
+logging.addLevelName(9,"AddINFO")  # Create a new level below DEBUG solely for additional info
+
+def AddINFO(self, message, *args, **kws):
+    if self.isEnabledFor(9):
+        self._log(9, message, args, **kws)
+
+logging.Logger.AddINFO = AddINFO
+# Create message through: logging.getLogger().AddINFO('YOUR MESSAGE')
+
```

Comment:
functions should not be in config file. Config file is strictly for definition of variables